app/main/helpers/direct_award_helpers.py

```diff
@@ -1,2 +1,22 @@
+from app import data_api_client
+
+
 def is_direct_award_project_accessible(project, user_id):
     return any([user['id'] == user_id for user in project['users']])
+
+
+def get_direct_award_projects(user_id):
```

Comment:
This should be called `get_direct_award_projects_by_status` or something like that, and the keys can just be `open` and `locked`?

I also think it's important that we at least acknowledge the `active` flag. I think it would be fine for inactive projects to go into a separate status bucket here e.g. `archived`, i.e. for them to not appear anywhere for now. We don't have a way of unsetting that 'active' flag anyway, so it's not like there should be any, but I can imagine someone deciding to set that flag by hand because 'reasons' in a support scenario :)